app/src/main/java/org/simple/clinic/setup/SetupActivityModule.kt

```diff
@@ -3,11 +3,26 @@ package org.simple.clinic.setup
 import dagger.Module
 import dagger.Provides
 import org.simple.clinic.AppDatabase
+import org.simple.clinic.BuildConfig
+import org.simple.clinic.appconfig.Country
 import org.simple.clinic.user.User
+import java.net.URI
+import javax.inject.Named
 
 @Module
 class SetupActivityModule {
 
   @Provides
   fun provideUserDao(appDatabase: AppDatabase): User.RoomDao = appDatabase.userDao()
+
+  @Provides
+  @Named("fallback")
+  fun providesFallbackCountry(): Country {
+    return Country(
+        isoCountryCode = "IN",
+        endpoint = URI.create(BuildConfig.FALLBACK_ENDPOINT),
```

Comment:
`URI.create(String)` can throw!

Reply:
Intentionally added.

Reply:
Since this is something we control completely, an invalid endpoint is not something the app should handle and should fail instantly.